### PR TITLE
refactor: Update page header for edit object page

### PR DESCRIPTION
**Pull request by raintygao**
### Description

Update page header for edit object page

## Screenshot
image
before

image
after

image
useNewHomePage disabled
## Testing the changes

Enable useNewHomePage and go to assets page.

## Changelog

- refactor: Update page header for edit object page

### Check List

- [ ] All tests pass
  - [ ] `yarn test:jest`
  - [ ] `yarn test:jest_integration`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [ ] Update [CHANGELOG.md](./../CHANGELOG.md)
- [ ] Commits are signed per the DCO using --signoff

### CODE REVIEW

**Comment by kamingleung** [comment]
Hi @raintygao, thanks for your update. A few items:
image

1. Let's remove the line between the banner and the container.
2. Is the yellow circle part of the pageheader component at the moment? We should remove that if it is custom for this page.

**Comment by raintygao** [comment]
> Hi @raintygao, thanks for your update. A few items: image
> 
> 1. Let's remove the line between the banner and the container.
> 2. Is the yellow circle part of the pageheader component at the moment? We should remove that if it is custom for this page.

Hi @kamingleung , this two items are controlled by header API and affect all pages, I think there will be a new PR latter to address this. cc @AMoo-Miki

**Comment by kamingleung** [comment]
@raintygao Got it. Looks good to me. Thanks.

**Comment by raintygao** [comment]
> @raintygao Got it. Looks good to me. Thanks.

Thanks Ming, for the first item I have reached out to @zhongnansu , for the second one this has been fixed after using latest OSD and security plugin code.

**Review by SuZhou-Joe — APPROVED** [review]

File: src/plugins/saved_objects_management/public/management_section/object_view/saved_object_view.tsx [hunk 1]
```diff
@@ -117,22 +122,56 @@ export class SavedObjectEdition extends Component<
...
+              <EuiSpacer size="s" />
+              <Intro />
+            </>
+          ))}
         {object && (
           <>
-            <EuiSpacer size="m" />
+            {useUpdatedUX ? (
+              <EuiTitle>
+                <h3>
```
Comment (ruanyl) [inline src/plugins/saved_objects_management/public/management_section/object_view/saved_object_view.tsx:165]: Should this be `h2` with `<EuiTitle size="s">`?

Comment (raintygao) [reply src/plugins/saved_objects_management/public/management_section/object_view/saved_object_view.tsx]: That looks more reasonable now, have updated in code and description. Thanks.

**Review by ruanyl — APPROVED** [review]

**Review by SuZhou-Joe — APPROVED** [review]

**Pull request merged**